phone-book/src/keypad.js

```diff
@@ -0,0 +1,308 @@
+const kuivstarIdentification = [
```

Comment:
please create a folder called keypad and make additional file for every class.

also, make constants(variables which would be never changed) in upper-case all letters, it's conversation at the define stage.

But I guess you have to create additional `JS-class`(probably it should be a service) for every phone-modifier